packages/editor-ui/src/components/NDVDraggablePanels.vue

```diff
@@ -295,6 +301,11 @@ export default Vue.extend({
 		onResizeEnd() {
 			this.storePositionData();
 		},
+		onResizeDebounced(data: { direction: string, x: number, width: number}) {
+			if (this.initialized) {
+				this.callDebounced('onResize', { debounceTime: 15, trailing: true }, data);
```

Comment:
Setting `debouceTime` to 15 makes the resizing _really_ "choppy", reducing it to `10` is way smoother and I don't the performance hit is going to be that sever, attaching GIFS of resizing using 15 compared to 10.
![CleanShot 2022-10-20 at 13 25 08](https://user-images.githubusercontent.com/12657221/196936406-1f95baee-3093-4d1a-bcac-6cfc282a8279.gif)
![CleanShot 2022-10-20 at 13 25 33](https://user-images.githubusercontent.com/12657221/196936416-e4f9de76-3f84-4de5-9f42-5d81b5b67f02.gif)